src/browser.ts

```diff
@@ -101,6 +132,20 @@ export class GrpcClient {
     );
   }
 
+  /**
+   * Browser version of createStub
+   * Creates a gRPC-fallback stub and auth
+   * @param {function} CreateStub - The constructor function of the stub.
+   * @param {Object} service - [TODO]
+   * @param {Object} options - The optional arguments to customize
+   *   gRPC connection. This options will be passed to the constructor of
+   *   gRPC client too.
+   * @param {string} options.servicePath - The name of the server of the service.
+   * @param {number} options.port - The port of the service.
+   * @param {grpcTypes.ClientCredentials=} options.sslCreds - The credentials to be used
```

Comment:
I believe we don't use `.sslCreds` in the code below, this line should be removed.